### PR TITLE
V1.4.1 finalise dg

### PR DESCRIPTION
**Pull request by CodingCookieRookie**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1379,13 +1379,36 @@ Deleting a student while only some students are being shown
...
+Adding a session while all sessions are being shown.
 
-{ end of `manual_testing#adding_a_session` written by: ________ }
+   1. Prerequisites: Multiple (but less than 100) sessions in the session list.
+ 
+   1. Test case: `addses s/Tutorial 6 d/12/7/2020`<br>
+      Expected: A session is added to the bottom of the session list. A success message including the particulars of the added student is shown in the result box.
+      
+   1. Test case: `addses s/Tutorial 7 d/25/12/2020`<br>
+      Expected: A session is added to the top of the session list. A success message including the particulars of the added student is shown in the result box.
```
Comment (erisjacey) [inline docs/DeveloperGuide.md]: Maybe this line could be changed to "Similar to previous"?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1379,13 +1379,36 @@ Deleting a student while only some students are being shown
...
+   1. Test case: `addses d/30/12/2020`<br>
+      Expected: Similar to previous.
+      
+   1. Test case: `addses s/Tutorial 6 d/25/7/2020`<br>
+      Expected: No session is added. Error message indicating that the session already exists is shown in the result box.
+        
+   1. Test case: `addses s/Tutorial 8 d/30/12/2020`<br>
+      Expected: No session is added. Error message indicating that the session already exists is shown in the result box.
+   
+{ end of `manual_testing#adding_a_session` written by: Alvin Chee Teck Weng}
```
Comment (erisjacey) [inline docs/DeveloperGuide.md]: Shouldn't there be a space between the last letter "g" and the last curly brace "}"?

**Review by erisjacey — APPROVED** [review]
LGTM, just a few nits

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (CodingCookieRookie) [reply docs/DeveloperGuide.md]: One should be top one should be bottom due to order of the date

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (CodingCookieRookie) [reply docs/DeveloperGuide.md]: Yup thanks!!

**Pull request merged**